multistreaming-server/Dockerfile

```diff
@@ -11,7 +11,7 @@ ARG PIPENV_PACKAGE_VERSION=2020.8.13
 RUN set -x \
  && addgroup -S stunnel \
  && adduser -S -D -H -h /dev/null -s /sbin/nologin -G stunnel -g stunnel stunnel \
- && echo "http://dl-3.alpinelinux.org/alpine/edge/testing/" >> /etc/apk/repositories \
```

Comment:
Let's use `latest-stable` rather than `v3.11` for the image version here, that way the build will track as the `jrottenberg/ffmpeg` base image gets updated. The original value of `edge` seemed like a good idea, but it seems that symlink is no longer supported in the source (it hasn't been updated since 2015).

Reply:
Thanks. see https://github.com/michaelkamprath/multi-service-rtmp-broadcaster/pull/39